### PR TITLE
create scan endpoint refactor

### PR DESCRIPTION
**Pull request by sunb26**
* fix "shadowed err" bazel build error
* refactor env reads to occur once at beginning instead of on request

### CODE REVIEW

File: handler/create_scan/create_scan.go [hunk 1]
```diff
@@ -71,13 +81,13 @@ func CreateScan(w http.ResponseWriter, r *http.Request) {
...
 	var scanId uint64
-	if err := insertScanRes.Scan(&scanId); err != nil {
+	if err = insertScanRes.Scan(&scanId); err != nil {
 		w.WriteHeader(http.StatusInternalServerError)
 		log.Printf("failed to insert into scan table: %s", err.Error())
 		return
 	}
 
-	if err := tx.Commit(); err != nil {
+	if err = tx.Commit(); err != nil {
```
Comment (Lev1ty) [inline handler/create_scan/create_scan.go:90]: this should be `:=`, weird it isn't

**Review by Lev1ty — APPROVED** [review]

File: handler/create_scan/create_scan.go (hunk 1 above)
Comment (sunb26) [reply handler/create_scan/create_scan.go:90]: That was actually what was causing teh bazel build to fail

image

Comment (Lev1ty) [reply handler/create_scan/create_scan.go:90]: try fixing line 44? looks like it can be inlined in the condition too

Comment (sunb26) [reply handler/create_scan/create_scan.go:88]: changed 44 and 58 to be inline but you get the same error becase err is assigned in line 71 still

**Review by Zack-Ren — APPROVED** [review]

**Pull request merged**